app/Models/Car.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Car extends Model
+{
+    use HasFactory;
+
+    protected $table = 'cars';
+
+    protected $primaryKey = 'id';
+
+    protected $fillable = [
+            'name',
+            'founded',
+            'description'
```

Comment:
```suggestion
            'description',
```
bổ sung dấu , ở phần tử cuối mảng, fix tương tự các chỗ khác